src/Test/Perf/util/test_util.csx

```diff
@@ -111,6 +111,16 @@ string ReleaseVbcPath()
     return Path.Combine(BinReleaseDirectory(), "vbc.exe");
 }
 
+string GetCPCDirectoryPath()
+{
+    return Environment.ExpandEnvironmentVariables(@"%SYSTEMDRIVE%\CPC");
```

Comment:
All of this go to `C:\CPC` and it is sandboxed there at this point. I dont really have a preference where it should go. If it turns out to be a pain point we can move it

Info about Directory path is helper so `test_util` seemed like a good place. If you have a preference then I can move it there.